bayesian/api/api_v2.py

```diff
@@ -248,6 +254,10 @@ def stack_analyses():
     if request.method == 'GET':
         raise HTTPError(400, error="Request id missing")
 
+    # Revert after RDS upgrade.
```

Comment:
Why are you returning error in SA Get?

Reply:
this was present from before.

Reply:
@mathur07 isn't it supposed to have uuid to show banner? or even it fails it will redirect to banner?

Reply:
@yzainee does it look fine to you? I am not sure if this is what we need.

Reply:
@rafiu007 I think we need to send a static id here.. right? @mathur07 can confirm

Reply:
I can return some dummy id.Or In UI we can for sometim hardcode a dummy id

Reply:
@mathur07 @yzainee ^^